lib/npm.js

```diff
@@ -70,8 +70,7 @@
 
   try {
     // startup, ok to do this synchronously
-    var j = parseJSON(fs.readFileSync(
-      path.join(__dirname, '../package.json')) + '')
+    var j = require(path.join(__dirname, '../package.json'))
```

Comment:
We don't use require for a reason (it provides poor error messages when there are errors in the file and I don't think it supports files starting with UTF-8 BOMs.

Reply:
> I don't think it supports files starting with UTF-8 BOMs.

it does

Reply:
but this is not important, the error message part makes perfect sense :) will undo